tools/java/org/unicode/cldr/api/CldrDraftStatus.java

```diff
@@ -0,0 +1,42 @@
+package org.unicode.cldr.api;
+
+import com.google.common.base.Ascii;
+import org.unicode.cldr.util.CLDRFile.DraftStatus;
+
+import java.util.Optional;
+
+/**
+ * Draft status for controlling which values to visit. Draft statuses are ordered by ordinal value,
+ * with {@code UNCONFIRMED} being the most lenient status (include everything) and {@code APPROVED}
+ * being the strictest.
+ */
+public enum CldrDraftStatus {
+    /** Include all values during visitation, performing no filtering. */
+    UNCONFIRMED(DraftStatus.unconfirmed),
```

Comment:
API locality again. Should we re-export `DraftStatus`? I'd prefer refactoring/renaming `DraftStatus` to move it here under api. This is exactly the kind of bifurcation I don't want to see with this effort.

Reply:
We could refactor the existing code to use this, though I'd say we should wait until the dust has settled for that. So far the duplication is only a few enums and a static method or two and it doesn't duplicate any business logic.